packages/app-mobile/components/ScreenHeader.tsx

```diff
@@ -196,34 +259,26 @@ class ScreenHeaderComponent extends React.PureComponent {
 		await Note.batchDelete(noteIds);
 	}
 
-	menu_select(value) {
+	private menu_select(value: OnSelectCallbackType) {
 		if (typeof value === 'function') {
 			value();
 		}
 	}
 
-	log_press() {
-		NavService.go('Log');
-	}
-
-	status_press() {
-		NavService.go('Status');
-	}
```

Comment:
These methods were unused (I did a search in all files in the `app-mobile/` directory).